pkg/client/jiva/controller_client.go

```diff
@@ -123,12 +123,12 @@ func (c *ControllerClient) ListReplicas(path string) ([]Replica, error) {
 // GetVolumeStats is used to get the status of volume controller.It is used to
 // get the response in json format and then the response is then decoded to the
 // desired structure.
-func (c *ControllerClient) GetVolumeStats(address string, obj interface{}) (int, error) {
+func (c *ControllerClient) GetVolumeStats(address string, api string, obj interface{}) (int, error) {
```

Comment:
replace api with - statsApiEndpoint

Reply:
yeah sure.

Reply:
@kmova The api argument is generic one. We are using it for "/stats" endpoint and "/replica" endpoint as well.So i feel api would be good

Reply:
ok.